### PR TITLE
Exercise3

### PR DESCRIPTION
**Pull request by SarahLM**
thanks for your review in advance ... 

our review for no-js-pls: https://github.com/Systems-Development-and-Frameworks/no-js-pls/pull/5#pullrequestreview-536799072

### CODE REVIEW

**Comment by FrauMauz** [comment]
changes in readme & in the description of the full request (added link to review for no-js-pls)

File: backend/src/_test_/posts.spec.js [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+            votes: 0,
+            author: { name: 'Test User' },
+          },
+        },
+      });
+    });
+  });
+
+  describe('VOTE_POST', () => {
+    let TEST_POST_ID;
```
Comment (TheKilljoy) [inline backend/src/_test_/posts.spec.js:137]: code convention: declaring a variable in caps means it's a constant and therefore should not be 'let'

**Review by TheKilljoy — APPROVED** [review]
Tests are fine, except one that is skipped, but we guess it will be fixed.
Code looks fine too. No runtime errors, no linter errors.

**Review by medizinmensch — APPROVED** [review]
Hey team! 
Very nice job :)
Nice

See my comments below.
You also included the indefinitely nestable queries, so  you get a ⭐ for that for sure.
I also see your review which I liked! ⭐ 

This brings you 10/10 ⭐ This was relatively rare for this tasks. So really really good!

**Pull request merged**